source/framework/core/src/TRestHits.cxx

```diff
@@ -545,103 +546,183 @@ void TRestHits::GetBoundaries(std::vector<double>& dist, double& max, double& mi
 Double_t TRestHits::GetGaussSigmaX() {
```

Comment:
It would be good to add a description to this method. Perhaps reusing some text from the PR description. I see this class is not following the standard headers. I will just quickly updated that

Reply:
Description added in [154df75](https://github.com/rest-for-physics/framework/pull/254/commits/154df759bffd7b34b65165775201da4290d087fb)